docker/ubuntu_22.04-dev.dockerfile

```
@@ -41,84 +36,38 @@ RUN apt-get update \
             future \
             progress
 
-FROM common_base AS conda_deps
-RUN apt-get update \
-    && apt-get install -y --fix-missing \
-        wget \
-        bzip2 \
-        ca-certificates \
-    && apt-get clean -y
-
-RUN echo 'export PATH=/opt/conda/bin:$PATH' > /etc/profile.d/conda.sh && \
-    wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda.sh && \
-    /bin/bash ~/miniconda.sh -b -p /opt/conda && \
-    rm ~/miniconda.sh
-
-ENV PATH /opt/conda/bin:$PATH
-
-RUN conda config --add channels conda-forge
-RUN conda update -n base -c defaults conda
-RUN conda install -y conda-libmamba-solver
-RUN conda config --set solver libmamba
-RUN conda install -y mamba
-RUN conda uninstall -y conda-libmamba-solver
-RUN conda config --set solver classic
-RUN conda update -y --all && \
-    mamba install -y \
-                cmake \
-                git \
-                libblas \
-                liblapack \
-                hdf5 \
-                setuptools \
-                pytest \
-                pytables \
-                jinja2 \
-                "cython<3" \
-                && \
-    mamba install -y --force-reinstall libsqlite && \
-    conda clean -y --all
-RUN mkdir -p `python -m site --user-site`
-
-FROM ${pkg_mgr}_deps AS base_python
 # make starting directory
 RUN mkdir -p $HOME/opt
 RUN echo "export PATH=$HOME/.local/bin:\$PATH" >> ~/.bashrc
 
 # build HDF5
-ARG build_hdf5="NO"
+ARG build_hdf5="hdf5-1_14_3"
 ENV HDF5_INSTALL_PATH=$HOME/opt/hdf5/$build_hdf5
-RUN if [ "$build_hdf5" != "NO" ]; then \
-        cd $HOME/opt \
-        && mkdir hdf5 \
-        && cd hdf5 \
-        && git clone --single-branch --branch $build_hdf5 https://github.com/HDFGroup/hdf5.git \
-        && cd hdf5 \
-        && ./configure --prefix=$HDF5_INSTALL_PATH --enable-shared \
-        && make -j 3 \
-        && make install \
-        && cd .. \
-        && rm -rf hdf5; \
-    fi
+RUN cd $HOME/opt \
+    && mkdir hdf5 \
+    && cd hdf5 \
+    && git clone --single-branch --branch hdf5-1_14_3 https://github.com/HDFGroup/hdf5.git \
```

shouild this line make use of the ARG build_hdf5 value
```suggestion
    && git clone --single-branch --branch $build_hdf5 https://github.com/HDFGroup/hdf5.git \
```